microapidev/auth-microapi
# feat: added adminUser create new password if he forgets password and …

### Overview 

- This PR is related to this **[issue](https://github.com/microapidev/auth-microapi/issues/377)**
- It contains the functionality for an admin user to create new password if they forget password by sending a password reset link to his email
- this password reset link verifies a token that is embedded in the url which can also expire. Adding a new password upon successful verification updates the adminUser's DB details with the new password that is hashed.

### How to test

- To test this make sure you run `npm install` to install all dependencies
- Run `npm run lint:fix` to fix all linting issues
- Test the endpoints: 
**`/api/admin/auth/forgot-password`** 
**`/api/admin/auth/reset-password/{token}`**
